src/plugins/intel_gpu/src/kernel_selector/kernels/broadcast/broadcast_kernel_base.h

```diff
@@ -35,8 +35,12 @@ class BroadcastKernelBase : public KernelBaseOpenCL {
 
 protected:
     JitConstants GetJitConstants(const broadcast_params& params) const;
-    static DispatchData SetDefault(const broadcast_params& params);
+    DispatchData SetDefault(const broadcast_params& params) const;
     KernelsData GetCommonKernelsData(const Params& params, const optional_params&) const;
     void GetUpdateDispatchDataFunc(KernelData& kd) const override;
+
+    // Tune params to the specific platform.
+    const size_t vec_size = 2;
+    const size_t y_blocks = 4;
```

Comment:
Seems that there is no reason that we need this variable in header. 
(Fixing something in the header file causes in the compilation of many files)
So if there is no specific reason, would like to suggest this to the cpp file.

Reply:
moved it.